app/src/main/res/values/strings.xml

```diff
@@ -16,4 +16,5 @@
     <string name="releaseDateLabel">Release Date</string>
     <string name="genresLabel">Genres</string>
     <string name="emptyDatabaseText">I have no data to show</string>
+    <string name="moreFragment">More</string>
```

Comment:
Wouldn't `moreFragmentLabel` be more accurate?

Reply:
I changed it